radio/src/translations/de.h.txt

```diff
@@ -325,7 +325,7 @@
 #define TR_VTMRMODES          			 "AUS""EIN""GSs""GS%""GSt"
 
 #define LEN_VTRAINERMODES              "\022"
-#define TR_VTRAINER_MASTER_OFF         "Off\0              "
+#define TR_VTRAINER_MASTER_OFF         "OFF\0              "
```

Comment:
Please use `AUS` instead of `OFF`.

It would be nice if you could change `Master` to `Lehrer` and `Slave` to` Schüler` just a little further down (Bluetooth / Multi).

Reply:
I'll do those now as I'm about to review this before merge...

Reply:
@pfeerick thx